persistence/remote/src/main/java/org/infinispan/persistence/remote/internal/RemoteStoreBlockHoundIntegration.java

```diff
@@ -0,0 +1,15 @@
+package org.infinispan.persistence.remote.internal;
+
+import org.kohsuke.MetaInfServices;
+
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+@MetaInfServices
+public class RemoteStoreBlockHoundIntegration implements BlockHoundIntegration {
+   @Override
+   public void applyTo(BlockHound.Builder builder) {
+      // TODO: this needs to be moved to the client hotrod module when it adds BlockHound
+      builder.allowBlockingCallsInside("org.infinispan.client.hotrod.impl.transport.netty.ChannelInitializer", "initSsl");
```

Comment:
In this PR?

Reply:
No, we should do that in a separate one. However I can add a URL to the JIRA (and create if there isn't one).